tests/TestModel.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
Is it necessary this class at this point?

Reply:
yes, for testing purposes